openwisp_network_topology/integrations/device/migrations/0002_create_relations.py

```diff
@@ -1,15 +1,6 @@
-import swapper
 from django.db import migrations
 
-
-def create_relations_0001(apps, schema_editor, app='topology'):
-    Node = swapper.load_model('topology', 'Node')
```

Comment:
using `swapper.load_model` gets the latest version of the model which might not be valid for this migration. [django docs](https://docs.djangoproject.com/en/3.1/topics/migrations/#data-migrations) advises us to use `apps` which gets the historic model up to the point of the migration. I experienced some delay in debugging reasons why some migrations were complaining the not existence of the newly added field because of this. 

I also moved this work to the `_init_.py` file for consistency.

Reply:
you're right, but the simulated model will not have the `auto_create` method and hence will fail anyway.
Please load the concrete `DeviceNode` model as it was before, Node can be simulated as you have done. The right solution for this problem is to switch to a management command, I created an issue for that: https://github.com/openwisp/openwisp-network-topology/issues/92